README.md

```diff
@@ -158,6 +159,20 @@ This installation type supports both Windows and Unix systems (Linux + MacOS). H
 
 If you are using macOS or other Linux operating systems you can jump the next section <a href="#installing-the-terminal">Installing the terminal</a>.
 
+### Web UI - Docker
```

Comment:
Can you add this on the table of contents above?

Reply:
Sorry, I forgot about the Table of Contents. One moment